gedi/gediFinder.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+"""gediFinder.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1PmKDBLkBxVhf1nrORDspAnqKCb7fkeMt
+
+# GEDI Finder script
+
+This script is meant to create a GediFinder URL and get the corresponding list of granules within a user-defined bounding box. This list can then be used in the Earthdata Search Tool to pull data from within a bounding box.
+"""
```

Comment:
Request: before merging, it probably makes sense to remove most of this auto-generated preamble that Colab throws in when exporting. Lines 2-10, that is; the description on L11 seems helpful to me.

Reply:
Just went through and cleaned up some of the commented stuff.